# DP: Description: Optimizes the build for sparcv9
# DP: Author: Ben Collins, Aurelien Jarno
# DP: Upstream status: Debian-Specific
# DP: Status Details:
# DP: Date: July 19, 2007

Index: configure
===================================================================
--- configure.orig	2007-10-23 08:46:15.000000000 -0400
+++ configure	2007-10-23 08:50:00.000000000 -0400
@@ -2306,6 +2306,11 @@
   machine="powerpc"
 fi
 
+# Force v9
+if test "$machine" = sparc; then
+   machine="sparcv9"
+fi
+
 # Braindead PowerPC box with absolutely no FPU.
 case "$machine-$host_os" in
   powerpc*-*soft)
Index: configure.in
===================================================================
--- configure.in.orig	2007-10-23 08:46:15.000000000 -0400
+++ configure.in	2007-10-23 08:50:00.000000000 -0400
@@ -307,6 +307,11 @@
   machine="powerpc"
 fi
 
+# Force v9
+if test "$machine" = sparc; then
+   machine="sparcv9"
+fi
+
 # Braindead PowerPC box with absolutely no FPU.
 case "$machine-$host_os" in
   powerpc*-*soft)
